src/Altinn.Platform/Altinn.Platform.Authorization/IntegrationTests/Util/AssertionUtil.cs

```diff
@@ -63,6 +63,28 @@ public static void AssertEqual(XacmlContextRequest expected, XacmlContextRequest
             AssertEqual(expected.Attributes, actual.Attributes);
         }
 
+        /// <summary>
+        /// Assert that two lists of <see cref="ResourcePolicy"/> have the same property values.
```

Comment:
Actually asserts that the lists have the same property values in the same position as well. Might be what we are wanting to archeive here. If this is the case, please update the comment.